pkg/control/controldisplay/formatter_test.go

```diff
@@ -1,263 +1,125 @@
-// 					{
-// 						Control: &c22,
-// 						Rows: []*controlexecute.ResultRow{
-// 							{
-// 								Status:     constants.ControlAlarm,
-// 								Reason:     "is pretty insecure",
-// 								Resource:   "some other resource",
-// 								Dimensions: []controlexecute.Dimension{},
-// 								Run:        &controlexecute.ControlRun{Control: &c22},
-// 							},
-// 						},
-// 					},
-// 				},
-// 			},
-// 		},
-// 	},
-// }
+type testCase struct {
+	input    string
+	expected interface{}
+}
 
-// const expectedJsonOutput = `{
-// 	"group_id": "DummyTest",
-// 	"title": "Test Root Group",
-// 	"description": "Description for test root group",
-// 	"tags": null,
-// 	"summary": {
-// 		"status": {
-// 			"alarm": 2,
-// 			"ok": 0,
-// 			"info": 0,
-// 			"skip": 0,
-// 			"error": 0
-// 		}
-// 	},
-// 	"groups": [
-// 		{
-// 			"group_id": "",
-// 			"title": "",
-// 			"description": "",
-// 			"tags": null,
-// 			"summary": {
-// 				"status": {
-// 					"alarm": 0,
-// 					"ok": 0,
-// 					"info": 0,
-// 					"skip": 0,
-// 					"error": 0
-// 				}
-// 			},
-// 			"groups": null,
-// 			"controls": [
-// 				{
-// 					"control_id": "",
-// 					"description": "",
-// 					"severity": "",
-// 					"tags": null,
-// 					"title": "",
-// 					"results": [
-// 						{
-// 							"reason": "is pretty insecure",
-// 							"resource": "some other resource",
-// 							"status": "alarm",
-// 							"dimensions": []
-// 						}
-// 					]
-// 				},
-// 				{
-// 					"control_id": "",
-// 					"description": "",
-// 					"severity": "",
-// 					"tags": null,
-// 					"title": "",
-// 					"results": [
-// 						{
-// 							"reason": "is pretty insecure",
-// 							"resource": "some other resource",
-// 							"status": "alarm",
-// 							"dimensions": []
-// 						}
-// 					]
-// 				}
-// 			]
-// 		},
-// 		{
-// 			"group_id": "",
-// 			"title": "",
-// 			"description": "",
-// 			"tags": null,
-// 			"summary": {
-// 				"status": {
-// 					"alarm": 0,
-// 					"ok": 0,
-// 					"info": 0,
-// 					"skip": 0,
-// 					"error": 0
-// 				}
-// 			},
-// 			"groups": null,
-// 			"controls": [
-// 				{
-// 					"control_id": "",
-// 					"description": "",
-// 					"severity": "",
-// 					"tags": null,
-// 					"title": "",
-// 					"results": [
-// 						{
-// 							"reason": "is pretty insecure",
-// 							"resource": "some other resource",
-// 							"status": "alarm",
-// 							"dimensions": []
-// 						}
-// 					]
-// 				},
-// 				{
-// 					"control_id": "",
-// 					"description": "",
-// 					"severity": "",
-// 					"tags": null,
-// 					"title": "",
-// 					"results": [
-// 						{
-// 							"reason": "is pretty insecure",
-// 							"resource": "some other resource",
-// 							"status": "alarm",
-// 							"dimensions": []
-// 						}
-// 					]
-// 				}
-// 			]
-// 		}
-// 	],
-// 	"controls": null
-// }`
+var formatterTestCase = []testCase{
+	{
+		input:    "bad-format",
+		expected: "ERROR",
```

Comment:
do we have test cases for alias?

Reply:
yes -  we have three dummy exporters (out of 5) which have an alias and is being tested.